### ISSUE TITLE
NTR: Chest imaging: pulmonary calcification (Lab: radiology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8918]
New term request Chest imaging: pulmonary calcification (elthat_100627221813): CXR; lung; Chest X-ray; thoracic  UMLS: 2073566

**Comment by pnrobinson** [comment]
@MickeySegal
it is probably worth specifying exactly what location is affected by calcification?

**Comment by MickeySegal** [comment]
https://www.ncbi.nlm.nih.gov/books/NBK448185/  "the initial granuloma may continue to grow in layers of inflammatory cells are deposited concentrically in an attempt to control the yeast. There is active inflammation in the periphery, and the central core may have calcification giving the appearance of malignancy. Frequently, a biopsy is done for these pulmonary masses to exclude malignancy"

Lung nodules https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9488618/ "As regards nodule morphological characteristics, besides small size, diffuse, central, laminated or popcorn calcifications, as well as fat tissue density and perifissural location have been recognised as indicative of benign lesions."

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000931
- label: Calcified pulmonary nodule
- parents: [Pulmonary nodule (HP:0033608)](https://hpo.jax.org/app/browse/term/HP:0033608)
- pmid: PMID:20582171
- def: A pulmonary nodule with calcium deposits. On radiography, these nodules appear dense.
- comment: Calcification in a pulmonary nodule (PN) on imaging indicates a high probability that the lesion is benign. But not all calcified PN are benign and the differential considerations include a primary central lung carcinoid, metastasis and a primary bronchogenic carcinoma.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8918
- synonyms:
